docs/index.md

```diff
@@ -2,6 +2,8 @@
 
 The AEA framework provides the tools for creating Autonomous Economic Agents (AEA). AEAs work continuously for your benefit without you having to do anything more than writing them and starting them up.
 
+<iframe width="560" height="315" src="http://www.youtube.com/embed/xpJA4IT5X88" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
If you make width `750` will align with the text :). Optional though.

Reply:
Well but that will depend on the device..

Reply:
Same for the other iframe. Totally optional.